doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -295,6 +295,74 @@ to change the `Issue tracker`_ status to "Resolved". Some issues may be
 flagged for backporting, in which case the status should be changed to
 "Pending Backport" (see the `Backporting`_ chapter for details).
 
+See also `Merging`_ for more information on merging.
+
+Proper Merge Commit Format
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is the most basic form of a merge commit::
+
+       doc/component: title of the commit 
+
+       Reviewed-by: Reviewer Name <rname@example.com>
+
+This consists of two parts:
+
+#. The title of the commits of the pull request to be merged.
+#. The name and email address of the reviewer. Enclose the reviewer's email 
+   address in angle brackets.
+
+Using .githubmap to Find a Reviewer's Email Address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you cannot find the email address of the reviewer on his or her github
+page, you can look it up in the **.githubmap** file, which can be found in
+the repository at **/ceph/.githubmap**.
+
+Using the Github API to Find a Reviewer's Email Address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the reviewer's email address is not in the **/ceph/.githubmap** file,
+use this procedure to find it in the Github API.
+
+#. Copy and paste this URL into your browser::
+
+        https://api.github.com/users/xxxxxxx/events/public
+
+#. Find the GitHub username associated with the target email address. You can
+   use my github username to test this procedure if you like. It's ``zdover23``.
+
+#. Replace the xxxxxxxx in the URL with the target username.
+
+#. Press Enter.
+
+#. Click ``Raw Data`` in the header bar. (This step is current and works as of 08
+   Aug 2020.)
+
+#. CTRL+F to search for the string ``"email":``  This includes the quotation marks
+   before and after the word ``email`` and also includes the colon.
+
+#. The target email address is to the right of the located search string.
```

Comment:
@zdover23 

There are many `email`s. So mention like:

Search inside `commits` array for author.name = "Zac Dover". The target email address is the value of (corresponding) author.email.

Reply:
I'm not sure this method is so reliable - it doesn't work for me or some other users I checked. It'd be better te leave it out imo.

Usually reviewers have committed something before, so you can find their email in 'git log' by searching for their name.